lib/Service/BotService.php

```diff
@@ -51,6 +54,36 @@ public function __construct(
 	) {
 	}
 
+	public function afterBotEnabled (BotEnabledEvent $event): void {
+		$this->sendAsyncRequest($event->getBotServer(), [
+			'type' => 'Join',
+			'actor' => [
+				'type' => 'Bot',
```

Comment:
According to https://www.w3.org/TR/activitystreams-vocabulary/#actor-types it should be Application I guess.
And we can add the id as well:
```suggestion
				'type' => 'Application',
				'id' => Attendee::ACTOR_BOTS . '/' . Attendee::ACTOR_BOT_PREFIX . $event->getBotServer()->getUrlHash()
```